src/editor/block/commands.js

```diff
@@ -50,14 +53,16 @@ export const addNewBlockAfterCurrent = ({ state, dispatch }) => {
 export function changeLanguageTo(state, dispatch, block, language, auto) {
     if (state.readOnly)
         return false
-    const delimRegex = /^\n∞∞∞[a-z]{0,16}(-a)?\n/g
+    const delimRegex = new RegExp(`\\n∞∞∞(${languageTokensMatcher})(-a)?(-c${timeMatcher})?(-u${timeMatcher})?\\n`, "g")
     if (state.doc.sliceString(block.delimiter.from, block.delimiter.to).match(delimRegex)) {
-        //console.log("changing language to", language)
+        const createdTimeStr = block.time.created || ""
+        const updatedTimeStr = block.time.updated ? newUpdatedTime() : ""
```

Comment:
This isn't explicitly needed - this will get updated by the `transactionFilter`, but it doesn't hurt to add it since the other function has the `transactionFilter` has the debounce